Documentation/ceph-cluster-crd.md

```diff
@@ -1592,13 +1592,14 @@ Communications will remain encrypted but the validity of the certificate will no
 
 For RGW, please note the following:
 
-* `VAULT_SECRET_ENGINE` option is specifically for RGW to mention about the secret engine which can be used, currently supports two: [kv](https://www.vaultproject.io/docs/secrets/kv) and [transit](https://www.vaultproject.io/docs/secrets/transit).
+* `VAULT_SECRET_ENGINE` option is specifically for RGW to mention about the secret engine which can be used, currently supports two: [kv](https://www.vaultproject.io/docs/secrets/kv) and [transit](https://www.vaultproject.io/docs/secrets/transit). And for kv engine only version 2 is supported.
+* The kubernetes secret name containin access token for vault kms need to specify in the `kmsTokenSecretName` of the [gateway spec of object store](ceph-object-store-crd.md#gateway-settings)
 * The Storage administrator needs to create a secret in the Vault server so that S3 clients use that key for encryption
 ```console
-# kv engine
+ # kv engine
 vault kv put rook/mybucketkey key=$(openssl rand -base64 32)
 
-# transit engine
+ # transit engine
 vault write -f transit/keys/mybucketkey exportable=true
```

Comment:
The new doc guidelines are to not use comments in `console` blocks. Additionally, if this is a command that users might need to modify (e.g. `mybucketkey`), then the lines should be prefixed by a shell prompt marker `$`

Example below

------
```console
$ vault kv put rook/<mybucketkey> key=$(openssl rand -base64 32)
$ vault write -f transit/keys/<mybucketkey> exportable=true
```